### ISSUE TITLE
9/29: Review RedCap How to Map on LucidChart

### ISSUE THREAD

**Issue by ritahitching** [lzim/teampsd#1374]
**Review RedCap How to Map on LucidChart**
Please review 

@staceypark 
- [ ]  RedCap How to Map on LucidChart - located [here](https://www.lucidchart.com/invitations/accept/baaa449c-2e9f-4ec5-af46-83ca80c4882f)
- [ ] I put a due date of 4/29 - but feel free to change
- [ ] I estimated a 1 for the review - but adjust accordingly 
- [ ] Cross reference for details of the issue task on #622 
- [ ] Lucidchart templates located [here](https://www.lucidchart.com/documents#/templates?folder_id=home) under **Personal** section.
- [ ] MTL.how map template [here](https://www.lucidchart.com/invitations/accept/def64d0c-2bda-4124-a7a4-c465ba3cb006)
- [ ] TeamPSD Bug & Feature template [here](https://www.lucidchart.com/invitations/accept/b8afc818-1a38-46c6-b985-04fea3081656) 
- [ ] ZenHub template [here](https://www.lucidchart.com/invitations/accept/777a8ae3-8992-4965-bd41-0d212a9ac038)
- [ ] VA RedCap diagram [here](https://www.lucidchart.com/invitations/accept/baaa449c-2e9f-4ec5-af46-83ca80c4882f)

**Comment by lzim** [comment]
Cross-ref #1378 

- @staceypark and @ritahitching Please see the **flow map** and **lucid chart** feedback that I provided on #1378.

Thanks! ⛵ 

>4. We need to **standardize flow chart symbols.**
>- I want to check whether the **Predefined Process** symbol for a "named process that is defined elsewhere" is more accurate than use of the **offpage connector**symbol, which implies the same process continues on another page https://www.zenflowchart.com/flowchart/
>- Edit maps to remove needless visual complexity, e.g., remove "steps" in connectors that could simply be a straight line.
>- NOTE: I also think there are also a number of other symbols (e.g., document, data, etc.) we should use to make our maps more easy to understand (e.g., the **RedCap** process needs these).
---

>5. A note on **Lucid Charts** Templates:
>- We should remove the VA Primary Dark Red #772432 from the Lucid Chart templates, as we did for all our Team PSD or _Modeling to Learn_ graphics. We have never used this color as it is very non-contemporatory. https://github.com/lzim/teampsd/blob/master/resources/maps/maps_markdown/zenhub_flow.md

>These steps will really clean up these resources 🧹

**Comment by staceypark** [comment]
@ritahitching  Thanks for your work to move this forward! 

1. I think there is confusion about a "**template**" vs. "**map**".
A **template** is a sample document with standardized pre-formatting (font, colors, shapes) to use as a starting point for new documents of the same type. 
A **map** is a document outlining the steps to a specific process, following the standard template format.

All of your "templates" are _**not**_ templates. Rather they are specific _**maps.**_
> * [ ]  MTL.how map template [here](https://www.lucidchart.com/invitations/accept/def64d0c-2bda-4124-a7a4-c465ba3cb006)
> * [ ]  TeamPSD Bug & Feature template [here](https://www.lucidchart.com/invitations/accept/b8afc818-1a38-46c6-b985-04fea3081656)
> * [ ]  ZenHub template [here](https://www.lucidchart.com/invitations/accept/777a8ae3-8992-4965-bd41-0d212a9ac038)

---
2. [This](https://www.lucidchart.com/documents/edit/cdbb3ad2-83ea-45ca-ad96-72322302ec3e/0_0) is the TeamPSD template that you created as part of the task #1139. This **template**, should include:
- Full list of VA colors
- Our standard shapes, meanings, and colors. Looks like you copied the documentation map as a start but never turned it into a generic template (meaning no specific map details should be included).
- Please review @lzim feedback from #1378 (and above) and make recommendations on what changes we need to make to our standard template.

---
3. Can you clarify what this Bug & Feature template is? I remember there was a task to create a checklist in #1378 but cannot find the requirements for a Bug & Feature map.
> * [ ]  TeamPSD Bug & Feature template [here](https://www.lucidchart.com/invitations/accept/b8afc818-1a38-46c6-b985-04fea3081656)

**Comment by lzim** [comment]
Thanks @staceypark 🔥 
This is absolutely helpful for clarifying the issue here:
> I think there is confusion about a "template" vs. "map".
A **template** is a sample document with standardized pre-formatting (font, colors, shapes) to use as a starting point for new documents of the same type.
A **map** is a document outlining the steps to a specific process, following the standard template format.

- **Maps** are helpful visual learning aids that define how a Team PSD process or Platform works.
- **Templates** #1374 or **Standard Style** #1364 helps to make the documents consistent in a way that makes them easy to use and learn from together in a manual.
- **Checklists** may be used to augment a **map** and help a Team PSD make sure they’ve completed the necessary steps - AND - all Team PSD checklists should follow a **standard style/checklist** across all Team PSD Platforms and Processs.
 
Therefore...
@ritahitching please cross-ref the notes I put on #1381, #1364, and #1192.
FYI: @staceypark @lijenn @branscombj @dlkibbe 

The key idea across the issues is the **ORDER OF OPERATIONS** 
- I would NOT continue doing these as unrelated discrete tasks.
- Rather, I would FINALIZE with Jane, Debbie and Jenn, the general principles that will guide all the other tasks.

In other words to get the **may_epic** cross-cutting priority to finish the **Team PSD Manual** #1192 and close all the open document cards in the **document_tracker**
1) finalize the **standardization rules** #1364 first, then once those are defined,
2) do the most overarching tasks that **further define Team PSD Project Management, or Team PSD Processes & Platforms,** like **GitHub** and our **Flow Maps** #1378 using those rules, and only then...
3) produce in a standard way the more **checklists and resources** that follow from 1) and 2) _meaning are style consistent with the overall manual, and follow the visual maps._

Does that make sense?

See the nice **Team PSD Manual** examples that I linked to on #1192 
*NICE Examples for Team PSD Manual:**
**Plain English for #1192**
https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/working-with-vsp/orientation/repo-guidelines.md

With Maps
https://github.com/department-of-veterans-affairs/lighthouse-facilities

**Comment by lzim** [comment]
Re-assigning to @ritahitching, since it’s on her plate. thanks!

**Comment by ritahitching** [comment]
@lzim please confirm I can move this issue to the May milestone &  Epic?
thanks

**Comment by lzim** [comment]
> @lzim please confirm I can move this issue to the May milestone & Epic?
> thanks

@ritahitching Right, consistent with my “order of operations” instruction above, which we reviewed in the **hq_workgroup** meeting, #1364 is first priority. Then, once finalized, all the dependent issue cards can proceed according to the standardization rules.

**Comment by ritahitching** [comment]
@lijenn this is issue is blocked by the #1192 and the MVP process.
If you read the thread you will find links to LucidChart and the most current flowmap.
I recommend we keep this issue open.

**Comment by lzim** [comment]
What about this one @lijenn  and @staceypark 

I think this is **oct_epic** now. Week 3 or week 4 should work for this.

**Comment by lijenn** [comment]
This is already done, just didn't close the card. Thanks!
